src/vos/tests/vts_common.h

```diff
@@ -36,6 +36,7 @@
 #endif /* FAULT_INJECTION */
 
 #define VPOOL_16M	(16ULL << 20)
```

Comment:
should we just remove 16M as a choice here?  And maybe internally only accept larger pools?  16MB is ridiculously small  even for pmdk